manifests/charts/ztunnel/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
how come we don't need this in any other chart?

Reply:
it's auto generated by `helm create`
IMHO, this's no bad, but I'm ok to remove these